bin/starter-pack/_main.php

```diff
@@ -9,11 +9,10 @@
  * Register the JS.
  */
 function add_extension_register_script() {
-
-	if ( ! class_exists( 'Automattic\WooCommerce\Admin\Loader' ) || ! \Automattic\WooCommerce\Admin\Loader::is_admin_page() ) {
+	if ( ! is_admin() ) {
```

Comment:
Not completely opposed to opening this up for all pages, but could we use `is_admin_or_embed_page()` to allow the nav slot fill to work where needed?

Reply:
oh yeah good idea. Can we be sure `is_admin_or_embed_page` is available to extensions? It isn't in my initial testing.

Reply:
If they register a page with the new navigation, they will qualify under `is_admin_or_embed_page()`.  Do you have an example extension that doesn't pass this test?  If so, we can keep `is_admin()`.

Reply:
Gotcha, thanks. I updated the code.